lib/integration/dbConnector.ts

```diff
@@ -0,0 +1,17 @@
+import { Client } from "pg";
+import { logMessage } from "../logger";
+
+export const dbConnector = async (): Promise<Client> => {
+  const connectionString: string | undefined = process.env.DATABASE_URL;
+  if (connectionString) {
+    logMessage.debug("Connexion initialization");
```

Comment:
small nit: Connection is spelled wrong

Reply:
This is a pros name from Clientconfig object that is passed to client.

Reply:
Keeping the same name in a log string will prevent creating an extra variable ? Not sure I understand. I mean `Connexion initialization` should be spelled `Connection initialization` and in fact it might be good to add detail here. `Postgres database connection initialization`

Reply:
I didn't get you were pointing at log string. It's been removed any.